### PR TITLE
Feat/add postgres spilo

### PR DESCRIPTION
**Pull request by Freebios**
This is a new Docker image that extends the Zalando Spilo image in order to install the extensions needed by Twenty : pg_graphql, wrappers, mysql_fdw

### CODE REVIEW

File: packages/twenty-docker/twenty-postgres-spilo/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+RUN apt update && apt install default-libmysqlclient-dev -y && rm -rf /var/lib/apt/lists/*
+
+RUN curl -L "https://github.com/supabase/wrappers/releases/download/v${WRAPPERS_VERSION}/wrappers-v${WRAPPERS_VERSION}-pg${POSTGRES_VERSION}-${TARGETARCH}-linux-gnu.deb" -o wrappers.deb && \
+    dpkg --install wrappers.deb && \
+    rm wrappers.deb
+
+COPY --from=build-libssl /usr/local/lib/libssl* /usr/local/lib/libcrypto*  /usr/lib/
+COPY --from=build-libssl /usr/local/lib/engines-1.1 /usr/lib/engines-1.1
+#RUN curl http://archive.ubuntu.com/ubuntu/pool/main/o/openssl/libssl1.1_1.1.0g-2ubuntu4_amd64.deb -o libssl.deb && dpkg -i libssl.deb && rm libssl.deb
```
Comment (Weiko) [inline packages/twenty-docker/twenty-postgres-spilo/Dockerfile:65]: Is this comment intended? 🙂

**Review by charlesBochet — APPROVED** [review]

File: packages/twenty-docker/twenty-postgres-spilo/Dockerfile (hunk 1 above)
Comment (Freebios) [reply packages/twenty-docker/twenty-postgres-spilo/Dockerfile:65]: Probably not :D

File: packages/twenty-docker/twenty-postgres-spilo/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,79 @@
+ARG POSTGRES_VERSION=15
+ARG SPILO_VERSION=3.2-p1
+ARG PG_GRAPHQL_VERSION=1.4.2
```
Comment (charlesBochet) [inline packages/twenty-docker/twenty-postgres-spilo/Dockerfile:3]: 1.5.1 should be better!

Comment (Freebios) [reply packages/twenty-docker/twenty-postgres-spilo/Dockerfile]: Indeed, solved

**Pull request merged**